powerauth-fido2/src/main/java/com/wultra/powerauth/fido2/rest/model/converter/serialization/AttestationObjectDeserializer.java

```diff
@@ -43,14 +45,28 @@ public class AttestationObjectDeserializer extends StdDeserializer<AttestationOb
 
     private final CBORMapper cborMapper = new CBORMapper();
 
+    /**
+     * Default deserializer constructor.
```

Comment:
Strictly speaking, this is not _default constructor_ (created by the compiler), but _no-arg_ created by you.

Reply:
Ah, correct, I will fix all of these.